gcp-py-oidc-provider-pulumi-cloud/README.md

```diff
@@ -0,0 +1,74 @@
+# Provisioning an OIDC Provider in Google Cloud for Pulumi Cloud
+
+This example will create OIDC configuration between Pulumi Cloud and Google Cloud, specifically demonstrating connectivity with [Pulumi ESC](https://www.pulumi.com/docs/pulumi-cloud/esc/). The program automates the process detailed in the Google Cloud documentation for the following activities:
+
+- [Create Workload Identity Provider and Pool](https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers#create_the_workload_identity_pool_and_provider)
+- [Authenticate the Workload](https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers#authenticate)
+
+## Prerequisites
+
+* [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+* [Configure Pulumi to Use GCP](https://www.pulumi.com/docs/clouds/gcp/get-started/begin/#configure-pulumi-to-access-your-google-cloud-account)
+* [Create a Google Cloud Project](https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers#configure) with the required APIs enabled
```

Comment:
Should we list the "required APIs" here for the sake of completeness?

Reply:
They are listed in the section that I linked. I wrote this in a way to try and minimize the amount of maintenance needed in the future in case those APIs ever change for some reason.

Reply:
Fair enough. I find the GCP documentation to be obtuse, at best, but it's reasonable to point to them. I'll mark this as resolved, but we should consider providing a bit more direct guidance on exactly where to find the list of required APIs.